example/main.go

```diff
@@ -2,7 +2,11 @@ package main
 
 import (
 	"fmt"
-	. "github.com/jondunning/gogoogledm"
+	"time"
```

Comment:
I guess this will be removed?

Reply:
the `time` ? It is used by the package for the field `timeToWait` (was already there).